cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -304,6 +312,83 @@ impl EdgeAddRemovePlugin {
 
         Ok(())
     }
+
+    fn add_conditional_edges(&self, graph: &mut cincinnati::Graph) -> Fallible<()> {
+        let mut conditional_edges: Vec<ConditionalEdge> = vec![];
+        graph
+            .conditional_edges
+            .iter()
+            .try_for_each(|ce| -> Fallible<()> {
+                let mut conditional_edge = ConditionalEdge {
+                    edge_regex: ce.edge_regex.clone(),
+                    edges: vec![],
+                    risks: ce.risks.clone(),
+                };
+
+                let to_regex_string = ce.edge_regex.to.clone();
```

Comment:
nitpick: we don't need to clone it here, strictly speaking - but I'm fine with it as a convenience

Reply:
fixed